libs/components/theme/src/lib/styles/_public-api/_compat/_mixins.scss

```diff
@@ -142,6 +142,7 @@
 
 @mixin sky-required-field {
   content: '*';
+  content: '*' / '';
```

Comment:
Asterisk still shows up on all the following browsers. Support for this way of hiding it from screen readers:
Mac
- Safari 
- Chrome ✅

Windows
- Chrome ✅
- Firefox 
- Edge ✅

Reply:
```
/* comment here */
content: '*';
content: '*' / '';
```